Formula/go-passbolt-cli.rb

```diff
@@ -0,0 +1,17 @@
+class GoPassboltCli < Formula
+  desc "CLI tool to interact with Passbolt, a Open source Password Manager for Teams"
+  homepage "https://github.com/passbolt/go-passbolt-cli"
+  url "https://github.com/passbolt/go-passbolt-cli/archive/refs/tags/v0.1.9.tar.gz"
+  sha256 "5787c350aa4a421cb7d811be7e2a465f0d447d5861172a7a0e6b54c2267d2f07"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
+  end
+
+  test do
+    system "#{bin}/go-passbolt-cli", "help"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Thanks for pointing out. This cli tool won't do anything without credentials and access to the server. Anyway, I've added two tests for configuration and for failing request.